pre_commit/git.py

```diff
@@ -173,7 +173,7 @@ def head_rev(remote: str) -> str:
 
 
 def has_diff(*args: str, repo: str = '.') -> bool:
-    cmd = ('git', 'diff', '--quiet', '--no-ext-diff', *args)
+    cmd = ('git', 'diff', '--quiet', '--no-ext-diff', '--no-textconv', *args)
```

Comment:
I don't think textconv is invoked for `--quiet`?

Reply:
Good point